x-pack/plugins/observability/server/lib/rules/slo_burn_rate/register.ts

```diff
@@ -37,7 +40,7 @@ export function sloBurnRateRuleType(createLifecycleRuleExecutor: CreateLifecycle
     },
     defaultActionGroupId: FIRED_ACTION.id,
     actionGroups: [FIRED_ACTION],
-    producer: 'slo',
+    producer: sloFeatureId,
     minimumLicenseRequired: 'basic' as LicenseType,
```

Comment:
@kdelemme Shouldn't this be `platinum`?

Reply:
You're right @CoenWarmer, I forgot that configuration when we changed the license requirements. It's not part of the scope of this PR though, @ymao1 I'll change in another PR, and thanks for reusing the `sloFeatureId` 👍🏻